docs/DeveloperGuide.md

```diff
@@ -378,234 +375,353 @@ Use case ends.
+    * 3b1. SellSavvy gives a warning that there are similar tags on the customer.
+
+  Use case resumes from step 4.
+
+**Use case 6: Add an Order under a Customer**
+
+* **Use Case**: UC06 - Add an Order under a Customer
+* **Preconditions**: There are orders displayed in the customer's list.
 * **Guarantees**:
     * A new pending order will be added under the specified customer, if input parameters are valid.
 
 **MSS**
 
-1. User wants to add an order under a specific customer.
-2. User finds the customer index from the list.
-3. User adds the customer using the index, specifying the details of the order.
-4. SellSavvy adds the order under the customer.
-5. SellSavvy confirms the addition by displaying the newly added order's details and customer's list of orders.
+1. User finds the customer index from the list.
+2. User adds the order by specifying the customer using the index and details of the order.
+3. SellSavvy adds the order under the customer.
+4. SellSavvy confirms the addition by displaying the newly added order's details and customer's list of orders.
 
 Use case ends.
 
 **Extensions**
 
-* 3a. SellSavvy detects required details missing.
-    * 3a1. SellSavvy displays an error message “Invalid command format!” and states the command format.
+* 2a. SellSavvy detects there are required parameters missing or the customer index is non-positive.
+    * 2a1. SellSavvy displays an error about the format and states the command format.
 
   Use case ends.
 
 
-* 3b. SellSavvy detects that there is a parameter not satisfying its constraint.
-    * 3b1. SellSavvy states the constraint of the invalid parameter.
+* 2b. SellSavvy detects that there is a parameter not satisfying its constraints.
+    * 2b1. SellSavvy states the constraints of the invalid parameter.
 
   Use case ends
 
 
-* 3c. SellSavvy detects that there are no customers with the specified index.
-    * 3c1. SellSavvy displays an error that the customer index is invalid.
+* 2c. SellSavvy detects that there are no customers with the specified index.
+    * 2c1. SellSavvy displays an error that the customer index is invalid.
 
   Use case ends.
 
 
-* 4a. SellSavvy detects that there is an existing pending order under the customer with a similar details.
-    * 4a1. SellSavvy gives a warning that an order with a similar details already exists.
+* 3a. SellSavvy detects that there is an existing pending order under the customer with a similar details.
+    * 3a1. SellSavvy gives a warning that an order with a similar details already exists.
 
-  Use case resumes from step 5.
+  Use case resumes from step 4.
 
-**Use case 6: List a Customer's Orders**
+**Use case 7: List a Customer's Orders**
 
-* **System**: SellSavvy
-* **Actor**: User
-* **Use Case**: UC06 - List a Customer's Orders
-* **Preconditions**: There are customers displayed in the customer list.
+* **Use Case**: UC07 - List a Customer's Orders
+* **Preconditions**: There are orders displayed in the customer's list.
```

Comment:
order list?

Reply:
why are orders displayed already when it is listorder?

Reply:
idk how this got changed by accident but ye should be that there are customers displayed in the customer list.